torch/ao/quantization/fx/prepare.py

```diff
@@ -119,12 +119,12 @@
 # note: the following default target dtype info dicts are temporary,
 # should be moved to the new programmable API class soon
 _DEFAULT_FP32_QCONFIG_FOR_TARGET_DTYPE_INFO = {
-    "input_act_obs_or_fq_ctr": torch.ao.quantization.qconfig._default_fp32_placeholder_qconfig.activation,
+    "input_obs_or_fq_ctr_for_args": torch.ao.quantization.qconfig._default_fp32_placeholder_qconfig.activation,
```

Comment:
Can we drop training `_for_args`? Also what does ctr stand for?

Reply:
Also there is not real "default", right? Because now each node can have different input structure. If this is meant for single tensor input qconfig, maybe name it as "default_fp32_tensor_qconfig.."

Reply:
I'm using `_for_args` since we have "_for_kwargs", would it be confusing if we drop it?

Reply:
`ctr` stands for constructor